routes/DesignAutomation.js

```diff
@@ -507,7 +527,20 @@ router.post('/forge/designautomation/workitems', multer({
     const inputFileNameOSS = `${new Date().toISOString().replace(/[-T:\.Z]/gm, '').substring(0, 14)}_input_${_path.basename(req.file.originalname)}`; // avoid overriding
     try {
         let contentStream = _fs.createReadStream(req.file.path);
-        await new ForgeAPI.ObjectsApi().uploadObject(bucketKey, inputFileNameOSS, req.file.size, contentStream, {}, req.oauth_client, req.oauth_token);
+        await new ForgeAPI.ObjectsApi().uploadResources(
+            bucketKey,
+            {
```

Comment:
This is certainly correct but I would suggest to pass an _array_ as the 2nd parameter to make it clear that the `uploadResources` method can upload multiple objects with a single call.